### ISSUE TITLE
Автлматический дробовик

### ISSUE THREAD

**Issue by Qwertyon666** [animusdev/green#395]
Итак, за основу берем АА-12
Думаю можно добавить один в арсенал, а так же куда-то там в изучение.
https://ru.m.wikipedia.org/wiki/AA-12

**Comment by Srankl1n** [comment]
нахуя нам второй боевой дробовик только с другим спрайтиком?

**Comment by Qwertyon666** [comment]
Ну не надо каждый раз передергивать затвор, да и в принципе можно запилить для него магазины, заряжаемые дробью

**Comment by Srankl1n** [comment]
>боевой дробовик >передергивать затвор. он же АВТОМАТИЧЕСКИЙ

**Comment by Qwertyon666** [comment]
Как знаешь

**Comment by NazarFogmann** [comment]
![image](https://cloud.githubusercontent.com/assets/19737119/21113880/3f5398f6-c0bc-11e6-8e51-846adab37dfb.png)
Найдите себя нормальных спрайтеров

**Comment by fluorescent-net** [comment]
1) АА12 - один из самых бездарных в плане дизайна стволов evah. Без нормального спрайта - просто пиздец.
2) У нас уже есть полуавтоматический дробовик. 
3) Слишком имбалансно.

Лучше бы такую няшу запилили для детектива. Винтаж, lever-action (задержку для перезарядки можно увеличить), 4+1 ёмкость. 
![](https://s-media-cache-ak0.pinimg.com/originals/48/5f/d2/485fd2b2a3e353692fe00ef8f300fd21.jpg)

**Comment by NazarFogmann** [comment]
![image](https://cloud.githubusercontent.com/assets/19737119/21117689/d0ddc332-c0cb-11e6-95c7-c956f47d24c6.png)
А в кэпскую на стеночку вот это повесим вот это охуенно

**Comment by Qwertyon666** [comment]
Может для каждого из глав сделать по оружию? Петы же есть

**Comment by ChristianAnderssen** [comment]
ты уже заебал, я даже не поленился зарегаться, чтобы сказать тебе об этом, у ХоСа, капитана и ХоПа есть по энергопистолету, у остальных телескопички, им хватит.

**Comment by ucnaHez** [comment]
Ячотнипонял. А чем вас не устраивает существующий автоматический дробовик?

**Comment by Qwertyon666** [comment]
Он полуавтоматический же

**Comment by ucnaHez** [comment]
Но у нас вообще нету автоматического оружия, концепт оружейной системы не тот

**Comment by Qwertyon666** [comment]
Есть, но не у сб

**Comment by ucnaHez** [comment]
Что? Где у нас в ССке автоматическое оружие вообще? Покажите мне это.

**Comment by silentenemy** [comment]
combat shotgun автоматический. Зажимаешь и ЛУПИШЬ ЕБАНОЙ ДРОБЬЮ ПО СРАНЫМ КСЕНОСАМ.
Реальная автоматика, да.

**Comment by ucnaHez** [comment]
@Qwertyon666 А ты на грине когда был последний раз? А комбатшотгана не надо дергать затвор. Каждый клик - выстрел. Кулдауна нет.

@silentenemy Ну не зажимаешь же. Просто каждый клик идет как выстрел.

**Comment by Qwertyon666** [comment]
Походу очень давно...

**Comment by silentenemy** [comment]
@ucnaHez Бля. А надо б реальную автоматику запилить, лол.

**Comment by Qwertyon666** [comment]
Что на счет персонального оружия для остальных глав?
Могу спрайты нарисовать

**Comment by Strinkor** [comment]
Еганов мало, разве? Да и не рационально огнестрел всем главам давать.

**Comment by Qwertyon666** [comment]
На морфлоте офицерам дают кортики, этим же можно дать пистолеты с закосом под их профу

**Comment by ucnaHez** [comment]
@Strinkor Да и без еганов обойдутся. У каждого главы в отсеке куча опасного говна.
@silentenemy И как ты себе это представляешь? С этим-то механом стрельбы кликом.

Я думал как-то об этом, даже раскопал бьендофункции, ловящие зажатие и отпуск мыши (юзается когда привязываешь там кого, раздеваешь), и думал запилить "Зажал - протащил через тайлы - отпустил - программа собрала тайлы по самому короткому пути между двумя точками - по выстрелу в каждую", но чего-то не дошло.

@Qwertyon666 Огнестрелу вообще на станции не место, он - оружие Синдиката. Тем более у боевых хоса и кепа есть пушка, остальные гражданские.

**Comment by Strinkor** [comment]
@ucnaHez Едиственное только - хоп без опасного говна, табуретом что разве уебет ну или шторками загрифонит.

**Comment by Qwertyon666** [comment]
Имеется ввиду аналог кэпского лазера + новые цели для триторов

**Comment by Qwertyon666** [comment]
В плане автоматической стрельбы можно подрубить прок: нажал кнопку на интерфейсе выскакивает окошко, туда вбиваешь кол-во пуль в очереди, 0 означает весь магазин, затем кликаешь по экрану и он в эту точку выпускает кол-во заданных пуль

**Comment by Strinkor** [comment]
Слишком долго. А если ты в бою? Пока ты вобьешь цифры - тебя уже баллоном вырубят.

**Comment by ucnaHez** [comment]
@Strinkor У потому у ХоПа еган.
@Qwertyon666 для целей можно вернуть ПОДШТАННИКИ
А вот эта идея - какая-то НИОЧЕНЬ. Во-первых, пока ты будешь открывать окошечко и вбивать кол-во пуль - тебя три раза уебут. А во-вторых - вот нахуя тебе МНОГО ПУЛЬ В ОДИН ТАЙЛ?

**Comment by Qwertyon666** [comment]
Что за подштанники? Вообще новая цель - это побочный эффект, а еган можно заменить.
Можно добавить ингейм модуль, который будет стрелять определенное кол-во раз, так как у каждого свой темп стрельбы, пускаю возятся с настройкой

**Comment by ucnaHez** [comment]
>а что за подштанники

Ты вообще в ССку играл? Триторы всегда воровали одежки глав.

Ну и этот "ингейм модуль" не решает главного вопроса: нахуя? Нахуя стрелять в одну точку много пуль?

**Comment by Qwertyon666** [comment]
А одежды...
Ну автоматическая стрельба повышает темп перестрелок, пока мы не можем сделать ее, то можно хотя бы частично повысить с помощью модуля/верба или вы под автоматом подразумеваете что если зажать мышку и водить ею по экрану, то вы будете обстреливать весь коридор беспрерывно?

**Comment by ucnaHez** [comment]
Да, в этом есть толк. А вот повышать темп перестрелок никто не будет, потому что хрен я навинчу на свой автомат с тридцатью пулями херню, которая количество выстрелов сократит до десяти (пусть в каждом по три пули, но толку-то? Уклонился - так ото всех разом, словил - так подойду и добью).

**Comment by Qwertyon666** [comment]
Идея по лучше: мы запускаем верб, создается только видимый нам объект - прицел. Наш персонаж ведет огонь в тайл, где находится прицел, пока мы этот прицел можем водить мышкой

**Comment by jammer312** [comment]
Я когда-то давно предлагал ввести разброс при стрельбе, который уменьшается, если периодически упражняешься в стрельбе (каждый раз, когда стреляешь), и постепенно увеличивается с ходом времени (но не выше какого-то порога).

Ввести рост разброса при каждом последующем выстреле в очереди.

Для автоматической стрельбы можно, как предлагал испанец, выбирать перетаскиванием мыши две точки, персонаж будет стрелять по тайлам между двумя этими точками, причем направление будет линейно меняться между двумя этими точками (сначала ведем ствол от точки А к точка Б, потом обратно, с постоянной по модулю скоростью), разброс будет постоянным, стрельба продолжается до момента, пока он не сменит активную руку, либо пока не прожмет резист, либо пока не стрельнет куда-либо еще.

**Comment by jammer312** [comment]
Тогда нет проблемы "зачем несколько выстрелов в один тайл", да и автоматическая и очередная стрельба становятся гораздо полезнее.
